dist/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Leaderboard</title>
+<script defer src="runtime.js"></script><script defer src="index.js"></script></head>
+<body>
+    <main>
+        <h1>Leaderboard</h1>
+        <section>
+            <div>
+                <h2>Recent Scores <button type="button">Refresh</button></h2>
+                <ul>
+                  <li>Name : 100</li>
+                  <li>Name : 45</li>
+                  <li>Name : 10</li>
+                  <li>Name : 80</li>
+                  <li>Name : 60</li>
+                  <li>Name : 50</li>
+                </ul>
+            </div>
+            <div class="second-div">
+              <h2>Add Your Score</h2>
+              <input type="text"><br>
+              <input type="text"><br>
+              <button class="submit" type="button">Submit</button>
+            </div>
```

Comment:
- [ ] Please add placeholders of "Your name" and "Your Score" to the "name" and "scores" input fields respectively as shown in the wireframe 🙏 
- [ ] It would be nice if you made the input `type` of the scores to equal number like so: `<input type="number" /> so that users may not submit text as scores 👍 
## Wireframe
![image](https://user-images.githubusercontent.com/46839548/195152926-5512267f-3e04-4654-a341-7d81ceb99991.png)
## Your Design
![image](https://user-images.githubusercontent.com/46839548/195153006-df8f914b-bf18-4bc8-9a08-f57709492aa7.png)